src/components/lib/pci/pci.c

```diff
@@ -0,0 +1,131 @@
+/* Simple PCI driver, only works for IVSHMEM device */
```

Comment:
Comment style is:

```c
/*
 * this is a typical multipline comment
 * that keeps on going.
 */
```
or 
```c
/***
 * for module-level documentation (like in your case).
 */
```

a double star at the start is for function-level documentation (e.g. `/** ... */`).